src/Tribe/Main.php

```diff
@@ -596,4 +597,31 @@ public function embed_head() {
 		<link rel="stylesheet" id="tribe-tickets-embed-css" href="<?php echo esc_url( $css_path ); ?>" type="text/css" media="all">
 		<?php
 	}
+
+	/**
+	 * Enqueue and localize script for use in ticket delete alert dialogue
+	 *
+	 * @since 4.3
+	 */
+	public function add_ticket_deletion_alert() {
+		$deletion_data = array(
+			'confirm_alert' => __( 'Are you sure you want to delete this ticket?', 'event-tickets' ),
+		);
+		tribe_asset(
+			self::instance(),
+			'tribe_tickets_ticket_delete_alert',
+			'ticket-delete-alert.js',
+			array( 'jquery' ),
+			'admin_enqueue_scripts',
+			array(
+				'localize' => array(
+					'name' => 'tribe_tickets_notices',
```

Comment:
This names the object as `tribe_tickets_notices` (tickets plural) but the script expects `tribe_ticket_notices` (ticket singular), which is why you're getting the undefined object notice you referenced in chat - so we just need to change that here (or in `ticket-delete-alert.js`, whichever you prefer), kill the commented out lines and the transition to `tribe_asset()` will be complete.